### DISCUSSION TITLE
SSL remove

### DISCUSSION THREAD

**Discussion by BBRCUG** [ant-media/Ant-Media-Server#6616] [General]
I have set up an SSL address. This no longer works. I can no longer access the web panel. How can I remove the SSL address again?

**Comment by muratugureminoglu** [comment]
Hi @BBRCUG 

Can you provide the following? 

- How did you activate SSL?
- Can you share logs (/usr/local/antmedia/log)
- Can you run netstat -tpln and share the output? 

If you disable the SSL completely, You should comment out the lines indicated in the link.

`https://github.com/ant-media/Ant-Media-Server/blob/master/src/main/server/conf/jee-container.xml#L67-L93`

The file is located `/usr/local/antmedia/conf/jee-container.xml`
